Kernel/PCI/Access.h

```diff
@@ -49,6 +49,7 @@ class PCI::Access {
     virtual ~Access() = default;
 
     Vector<PhysicalID> m_physical_ids;
+    Vector<u8> m_enumerated_buses;
```

Comment:
It looks like we might be able to make this a `AK::Bitmap` instead given how it's currently used?